### ISSUE TITLE
Missing empty srs

### ISSUE THREAD

**Issue by kcotar** [connormanning/entwine#268]
When building data without any projection info, version `2.1` produced an empty srs entry in `ept.json`. With version `2.2` this field is not present anymore, and data can not be visualised with Potree without manually adding the srs field.

**Comment by defaultbranch** [comment]
@kcotar, I observed a similar behavior, but I don't consider it a bug of entwine. For me, the current behavior (to break Potree when no reference data is set) is actually helpful, it prevents mixing up global-vs-local coordinate systems.

*Reproduction* (just for clarification)

Required: some `$WORKDIR` folder, containing some `$LASFILE`, with no embedded information on the coordinate system; the following steps try to run `entwine build -i $WORKDIR/$LASFILE -o $WORKDIR/output/` in a recent container image:

- `podman pull docker.io/connormanning/entwine:sha-41d37a3f` (put `docker` instead of `podman`, if desired)
- `podman run -it --rm --mount type=bind,source=$WORKDIR,target=/workdir entwine build -i /workdir/$LASFILE -o /workdir/output`
- then (as you did), use Potree to display the EPT (ept.json+…)

*Outcome*

Potree logs an error to the Javascript console, reporting the missing srs info.

*Solution* (which works for me)

Run `entwine … --srs "EPSG:2056"… ` (this is Switzerland, use the EPSG appropriate code); this helps to label the data's reference system and to avoid problems when working with multiple reference systems.

**Comment by connormanning** [comment]
Yeah, agree that this is out of scope for Entwine and is a one-line fix upstream in Potree.
